jwst/tests_nightly/general/fgs/test_fgs_image2_1.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from jwst.pipeline.calwebb_image2 import Image2Pipeline
+
+from jwst.tests.base_test import BaseJWSTTest
+
+
+@pytest.mark.bigdata
+class TestImage2Pipeline(BaseJWSTTest):
+    input_loc = 'fgs'
+    ref_loc = ['test_image2pipeline', 'truth']
+
+    def test_fgs_image2pipeline1(self, _jail):
```

Comment:
This is already inherited, so I don't think you need to re-declare `_jail` here. This comment also applies to all similar occurrences.